PendingReleaseNotes.md

```diff
@@ -13,3 +13,4 @@
 - Added experimental `cephConfig` to CephCluster to allow setting Ceph config options in the Ceph MON config store via the CRD
 - CephCSI v3.10.0 is now the default CSI driver version.
   Refer to [Ceph-CSI v3.10.0 Release Notes](https://github.com/ceph/ceph-csi/releases/tag/v3.10.0) for more details.
+- Automate the faster recovery of the CephFS RWO volume from node loss utilizing kubernetes feature [Non-Graceful Node Shutdown](https://kubernetes.io/blog/2022/12/16/kubernetes-1-26-non-graceful-node-shutdown-beta/) by requiring manual tainting of the node with an 'out-of-service' taint once admin confirm that the node is down. This feature also prerequisites on the [CSI-addons](https://rook.github.io/docs/rook/latest/Storage-Configuration/Ceph-CSI/ceph-csi-drivers/?h=csiaddons#csi-addons-controller) for a network fencing CRD, enable `csi-addons` sidecar and the minimum required Kubernetes version is v1.26.
```

Comment:
Do we expect anyone to use RWO volumes with CephFS? Rook users would really use rbd volumes with RWO. Maybe they will choose to use it, I just want to be clear if there is a scenario for Rook users.

Reply:
After discussing with Niels, one scenario can be backing up a snapshot.
RWO with CephFS is supported, but users are recommended to use RBD for RWO volumes